### DISCUSSION TITLE
DTSU666-H Registers over Modbus TCP (SDongle)

### DISCUSSION THREAD

**Discussion by mgherzan** [wlcrs/huawei_solar#635] [General]
Please, can someone confirm that the registers/data from a DTSU666-H meter are available over Modbus TCP (as per section 3.3 of the "Solar Inverter Modbus Interface Definitions" document) and presented in Home Assistant by the integration?

Please assume a simple setup: the smart meter is directly connected to the inverter.

**Comment by achtbitunix** [comment]
Hi,
the power meter is there.
![Screenshot 2024-02-28 at 08 12 58](https://github.com/wlcrs/huawei_solar/assets/46343677/01f7cafc-7be5-458d-b505-d9a8e88a5fd0)
Running version 1.2.6b5
